Java/ds/Reverse_alternate_k_Nodes.java

```diff
@@ -0,0 +1,127 @@
+﻿//Given the head pointer to a linked list of length n and an integer k, we have to reverse every alternate k nodes in the linked list.
+
```

Comment:
Add a bit more meaningful description

Reply:
@ankitaggarwal23 Should I add a sample input/output in the description to make it more comprehensible ?

Reply:
No you can add your approach or may be if you used any algo you can add a bit more about algo etc.